portfolio/.classpath

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure we should include these .classpath, .project and .settings files in the repo. They seem to be IDE metadata, and I think they clutter the code a little. Apparently they can also make it harder to work with this repo using a different IDE.

In saying that, opinions seem to be split as to whether to include this metadata ([1](https://softwareengineering.stackexchange.com/questions/202051/what-should-i-include-in-my-repositiory-from-ide-projects), [2](https://stackoverflow.com/questions/17024379/how-to-manage-ide-files-in-a-git-repository)). Thoughts @jupi3200 ?

If we decide to not include them, you can get Git to ignore them with [a .gitignore file](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files).

Reply:
+1

Reply:
That makes sense. Thank you for pointing this out! I have already updated the .gitignore file, so those setting files are not expected to appear again. Sorry that I was trying to fix the package name in walkthroughs example, but then add the massive setting files for almost all example directories. And even after adding .gitignore, it can only avoid future actions, and I need to delete those files manually one by one on GitHub website. Sorry for the massive notification emails!